sessionctx/variable/tidb_vars.go

```diff
@@ -189,9 +192,11 @@ const (
 	DefTiDBRetryLimit                = 10
 	DefTiDBHashJoinConcurrency       = 5
 	DefTiDBOptimizerSelectivityLevel = 0
+	DefTiDBDDLReorgWorkerCount       = 16
 )
 
 // Process global variables.
 var (
-	ProcessGeneralLog uint32
+	ProcessGeneralLog     uint32
+	DDLReorgWorkerCounter int32 = DefTiDBDDLReorgWorkerCount
```

Comment:
s/DDLReorgWorkerCounter/ddlReorgWorkerCounter ?
We use `GetDDLReorgWorkerCounter` and `SetDDLReorgWorkerCounter` to change its value. So I think we'd better use the lowercase variable.